MultimodalQnA/ui/gradio/multimodalqna_ui_gradio.py

```diff
@@ -109,7 +119,7 @@ def http_bot(state, request: gr.Request):
             timeout=100,
         )
         print(response.status_code)
-        print(response.json())
+        # print(response.json())
```

Comment:
These prints can be useful, but I commented them out because they can add lots of time to each chat update or even cause them to time out.